client/sound-system.hpp

```diff
@@ -7,6 +7,7 @@
 #include <memory>
 #include <set>
 #include <iostream>
+#include <thread>
```

Comment:
Is this necessary?

Reply:
There is a call to `std::this_thread` in the update loop to sleep, so yes. Also the build fails without it now for some reason that it worked before 😞